src/Routing/Route/DashedRoute.php

```diff
@@ -56,11 +56,12 @@ protected function _camelizePlugin($plugin)
      * camelBacked form.
      *
      * @param string $url The URL to parse
+     * @param string $method The HTTP method.
      * @return array|false An array of request parameters, or false on failure.
      */
-    public function parse($url)
+    public function parse($url, $method = '')
```

Comment:
@markstory I think so. At least, I feel ambiguous about the current wording. What about showing concrete examples?
Aside from that, if the line means `adding a default value to an already existing argument`, then I am OK about this change. 
But wouldn't it better passing `$context = []` than `$method = ''`? We might want to add another context value again though.